lib/fog/compute/google/models/disk.rb

```diff
@@ -74,11 +74,15 @@ def attached_disk_obj(opts = {})
         end
 
         def get_as_boot_disk(writable = true, auto_delete = false)
+          get_disk(writable, true, auto_delete)
+        end
+
+        def get_disk(writable = true, boot = false, auto_delete = false)
```

Comment:
Since we already have `disks.get`, I think it'll make more sense to name this `get_as_hash` or something similar to avoid confusion, WDYT?

Reply:
As there is a `get_as_boot_disk`, how about `get_as_disk`?